Update qdk-relnotes.md (#132)
MicrosoftDocs/quantum-docs: articles/qdk-relnotes.md

For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).

> ## Version 0.15.2103.133969

Author: ```suggestion
## Version 0.16.xxxx.xxxxxx
```

*Release date: April 22nd, 2021*

> - HM type inference algorithm.
> - Added support for empty array literals.

Reviewer: ```suggestion
- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
```

For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).

## Version 0.16.xxxx.xxxxxx

*Release date: April 22nd, 2021*

- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.

> - Added support for more types for 'w' and 'indices' inputs for QIO Python SDK.

Reviewer 2: I would rephrase this as:
"Added support for NumPy types in coefficient definitions for problems in QIO Python SDK."

> - Updated control-plane CLI with new swagger to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).

Reviewer 2: Also, recommend changing this to "Updated control-plane swagger file to [support ...](...)"
Let me prepare a separate PR for the CLI items to include past ones as well.

For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).

## Version 0.16.xxxx.xxxxxx

*Release date: April 22nd, 2021*

- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
- Added support for NumPy types in coefficient definitions for problems in QIO Python SDK.

> 

Author: ```suggestion
```

For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).

## Version 0.16.xxxx.xxxxxx

*Release date: April 22nd, 2021*

- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.

> 

Reviewer 2: NIT: No need to separate in sections with the blank lines, we can keep everything together.

> 

Author: ```suggestion
```

- Added support for NumPy types in coefficient definitions for problems in QIO Python SDK.

> - Released v0.X.0 of the Az CLI extension: Updated control-plane swagger file to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).

Reviewer 2: I'll send a PR into your PR addressing the CLI updates, we can discuss there.

> - Released v0.X.0 of the Az CLI extension: Updated control-plane swagger file to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).

Reviewer 2: ```suggestion
- Updated control-plane swagger file to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```

> - Released v0.X.0 of the Az CLI extension: Updated control-plane CLI with new swagger to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).

Author: ```suggestion
- Released v0.X.0 of the Az CLI extension: Updated control-plane swagger file to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```
